build.sbt

```diff
@@ -367,7 +376,7 @@ lazy val observable = Project(id = "observable", base = file("modules/observable
 lazy val common = Project(id = "common", base = file("modules/common"))
   .dependsOn(observable, sbtDependencyManager)
   .settings(
-    version  <<= (version in ThisBuild, sparkVersion) { (v,sv) => s"${sv}_${v}" }
+    version := s"${version.in(ThisBuild).value}_${sparkVersion.value}"
```

Comment:
it used to be `$sparkVersion_$version`.
need to invert order, for snapshots not to be messed up in Maven repo!!

actually, we spotted a tiny inconsistency with https://github.com/spark-notebook/spark-notebook/pull/915/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceL398 which was inverted..

how about extracting a val here too, e.g. `versionShortWithSpark` ?

see https://gist.github.com/james64/256ee66deea6f0ab1efd2b40f465b6c7#file-versions-comparison-sbt-log-L52